client/src/components/header/Header.svelte

```diff
@@ -0,0 +1,83 @@
+<script>
+  import { slide } from 'svelte/transition';
+  let show = false;
+  function showOptions(){
+    show = show==false ? true: false
+  }
+</script>
+
+<main>
+  <div class="container">
+    <div class="title">Step By Step</div>
+    
+      <img on:click={showOptions} on:keydown={showOptions} class="profileIcon" alt="profileIcon" src="https://cdn-icons-png.flaticon.com/512/6522/6522516.png" />
```

Comment:
wrap this thing in a button tag, add on:click to the button so you don't need to add on:keydown